frontend/src/components/DefenceBox/DefenceMechanism.tsx

```diff
@@ -72,35 +79,49 @@ function DefenceMechanism({
 		>
 			<summary>
 				<span aria-hidden>{defenceDetail.name}</span>
```

Comment:
Interestingly, "prompt enclosure" isn't read out by the narrator when it's focused. I think it's because the checkbox is required for that. Just tried it out and one solution is to not hide the name when the defence is prompt enclosure.

```suggestion
				<span aria-hidden={defenceDetail.id !== DEFENCE_ID.PROMPT_ENCLOSURE}>
					{defenceDetail.name}
				</span>
```